how-to/software/snapshot-service.md

```diff
@@ -0,0 +1,284 @@
+(snapshot-service)=
+# How to use the Ubuntu snapshot service
+
+The Ubuntu snapshot service allows you to easily access old packages in the
+Ubuntu archive based on past dates. Some of the use cases for this service
+include
+
+* Installing a superseded version of a package as at a particular date and
+  time, which is useful to troubleshoot bugs or regressions or to allow
+  reproducibility.
+* Setting upgrades to known states, which can be useful to ensure homogeneity
+  for a fleet of Ubuntu installations and to ensure predictability upon package
+  upgrades (including unattended upgrades).
+* Support a structured update workflow by validating snapshots in different
+  environments or by rolling out package upgrades in stages for a large set of
+  machines.
+
+This how-to guide shows how to set-up and use the snapshot service.
+
+## Prerequisites
+
+Snapshots are supported in Ubuntu 23.10 onwards and on updated installations of
+Ubuntu 20.04 LTS (starting from `apt` 2.0.10) and Ubuntu 22.04 LTS (starting
+from `apt` 2.4.11).
+
+In Ubuntu 24.04 LTS and later, `apt` will automatically detect whether a
```

Comment:
suggestion (non-blocking): Consider adding a table of the different supported versions and their remarks.

Reply:
+1 I think a small table will help provide clarity. I'm not sure what "updated installations of Ubuntu 20.04 and 22.04" means - what's the thing that needs to be updated?